Android.bp

```diff
@@ -341,4 +341,126 @@ cc_library_static {
     },
 }
 
-subdirs = ["test"]
+cc_library_static {
+    name: "libxaacenc",
+
+    vendor_available: true,
+    host_supported: true,
```

Comment:
As the current build has different cflags for few platforms, consider adding them using "arch"